README.md

```diff
@@ -101,7 +102,8 @@ Example commands:
 
```

Comment:
![image](https://github.com/BHObserver/To-Do-List/assets/93039359/5fc9dacb-2d5e-4b3c-b44b-92a957e61507)

**[OPTIONAL]**

- [ ] It would be nice if you update your readme **title** and **description** to make your readme descriptive and professional 👍